tequilapi/doc.go

```diff
@@ -15,7 +15,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-// Tequila API
+// Package tequilapi Tequila API
```

Comment:
can we add a proper description here since we're already changing it?

Reply:
What do you call proper description? This change was made to make golint happy.

Reply:
Something along the lines of `Package tequilapi contains the Tequila API http server as well as a client for interacting with it`

Reply:
I wonder how it look in swagger docs. As this first line is important for godoc and for swagger too.